package.json

```diff
@@ -29,6 +29,9 @@
   },
   "author": "Sebastian McKenzie <sebmck@gmail.com>",
   "license": "MIT",
+  "engines": {
+    "node": ">=4"
```

Comment:
If a new node version breaks, we have to take action in any case. If it not breaks, then in this case we wouldn't have to do anything, but with limit we would always need to care. So I think it is fine like it is :)